app/views/idv/phone/new.html.erb

```diff
@@ -65,6 +65,25 @@
         required: true,
         class: 'margin-bottom-4',
       ) %>
+  <%= render AlertComponent.new(
+        type: :warning,
+        class: 'margin-bottom-4',
+        text_tag: 'div',
+        id: 'phone-already-submitted-alert',
+        data: {
+          failed_phone_numbers: @idv_form.failed_phone_numbers,
+        },
+        hidden: true,
+      ) do %>
+    <%= t('idv.messages.phone.failed_number.alert_text') %>
+    <%= gpo_letter_available && t(
+          'idv.messages.phone.failed_number.gpo_alert_html',
+          link_html: link_to(
+            t('idv.messages.phone.failed_number.gpo_verify_link'),
+            idv_gpo_path,
+          ),
+        ) %>
```

Comment:
We should use an expanded `if` block here, otherwise the literal "false" is rendered:

![image](https://github.com/18F/identity-idp/assets/1779930/10f00bb4-713b-4ba2-b4bf-06d62d26064f)

```suggestion
    <% if gpo_letter_available %>
      <%= t(
            'idv.messages.phone.failed_number.gpo_alert_html',
            link_html: link_to(
              t('idv.messages.phone.failed_number.gpo_verify_link'),
              idv_gpo_path,
            ),
          ) %>
    <% end %>
```